nix/upm/imported-packages

```diff
@@ -0,0 +1,46 @@
+archive/tar
```

Comment:
My recommendation for this would be to have a github action that just updates this file after pushes to master automatically. After all it's a performance optimization and it causes too much friction/merge conflicts to have this being updated in pull requests.

Reply:
Something like that but for updating this file: https://gist.github.com/Broxzier/1ed980b8b3822d213e98042fc6a92040